de.metas.printing/de.metas.printing.base/src/main/java/de/metas/printing/api/impl/PrintJobBL.java

```diff
@@ -191,104 +170,17 @@ public int createPrintJobs(final IPrintingQueueSource source,
 		}
 		finally
 		{
-			batchMonitor.finish();
-			enqueueForPDFPrinting(source, monitor, pdfPrintingJobInstructions, printJobCount);
-			
+			final PrintingAsyncBatch printingAsyncBatch = PrintingAsyncBatch.builder()
```

Comment:
I wasn't aware of this enqueueForPDFPrinting thing before, but now i'm wondering: what does it do? 
looking at the javadoc in IPrintJobBL, i don't see why it should **always** do something about the PDF printing.

how does it work for users that use physical printers?
feel free to also contact me by voice too, but i think in the end there needs to be something written, either by clearer code or by comments